### PR TITLE
Fix  --version option is not working properly #1351

### PR DESCRIPTION
**Pull request by chellem**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes 
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | 1351

### CODE REVIEW

**Comment by antonmedv** [comment]
Please add comments about implementation,

File: bin/dep [hunk 1]
```diff
@@ -86,18 +86,25 @@ set_include_path($includePath . PATH_SEPARATOR . get_include_path());
...
-
-$composerLockFile = $deployFilePath . '/composer.lock';
-if (is_readable($composerLockFile)) {
-    $lock = json_decode(file_get_contents($composerLockFile), true);
-    foreach ($lock['packages'] as $package) {
-        if ($package['name'] === 'deployer/deployer') {
-            $version = $package['version'];
-        }
+$composerLockFiles = array(
+        realpath($deployFilePath . '/../composer.lock'),
```
Comment (antonmedv) [inline bin/dep]: Why ..?

File: bin/dep [hunk 2]
```diff
@@ -86,18 +86,25 @@ set_include_path($includePath . PATH_SEPARATOR . get_include_path());
...
+        realpath(__DIR__ . '/../../../../composer.lock')
```
Comment (antonmedv) [inline bin/dep]: ../../../..?

**Comment by chellem** [comment]
@antonmedv the changes doesn't work?

File: bin/dep (hunk 2 above)
Comment (chellem) [reply bin/dep]: getting the global composer.lock

File: bin/dep (hunk 1 above)
Comment (chellem) [reply bin/dep]: because I don't find composer.lock in the same directory as $deployFilePath

**Comment by antonmedv** [comment]
But it does working now: 
```
php vendor/bin/dep --version
```

**Comment by antonmedv** [comment]
After a lot fo thinking, I decided to go away from using version detection. Now version will be set manually every time before release.

**Pull request closed without merging**